src/SoftKiller.jl

```diff
@@ -0,0 +1,203 @@
+using JetReconstruction
+
+"""
+SoftKiller
```

Comment:
This should be indented 4 spaces and be the struct signature:

```jl
   mutable struct SoftKiller
```

The more descriptive parts of this docstring about FastJet should be in the package documentation.